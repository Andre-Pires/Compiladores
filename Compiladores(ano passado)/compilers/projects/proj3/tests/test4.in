b = b + 1
a = b + c
c = c + 2
d = c * 2
e = a + b
