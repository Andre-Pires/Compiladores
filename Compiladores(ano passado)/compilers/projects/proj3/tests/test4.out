 Number of Registers: 6
	r4 = ARP + 8
	r4 = *r4
	r4 = r4 + 1
	r3 = ARP + 12
	r3 = *r3
	r2 = r4 + r3
	r3 = r3 + 2
	r1 = r3 *2
	r0 = r2 + r4
	r5 = ARP + 20
	*r5 = r0
	r5 = ARP + 16
	*r5 = r1
	r5 = ARP + 4
	*r5 = r2
	r5 = ARP + 12
	*r5 = r3
	r5 = ARP + 8
	*r5 = r4
 Number Load Operations: 2
 Number Store Operations: 5
